ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -404,47 +404,11 @@ public void releaseResourcesTest() throws JSONException, UnsupportedEncodingExce
    */
   @Test
   public void postGetHeadDeleteTest() throws Exception {
-    final int CONTENT_LENGTH = 1024;
-    ByteBuffer content = ByteBuffer.wrap(TestUtils.getRandomBytes(CONTENT_LENGTH));
-    String serviceId = "postGetHeadDeleteServiceID";
-    String contentType = "application/octet-stream";
-    String ownerId = "postGetHeadDeleteOwnerID";
-    JSONObject headers = new JSONObject();
-    setAmbryHeadersForPut(headers, 7200, false, serviceId, contentType, ownerId, refAccount.getName(),
-        refContainer.getName());
-    Map<String, String> userMetadata = new HashMap<>();
-    userMetadata.put(RestUtils.Headers.USER_META_DATA_HEADER_PREFIX + "key1", "value1");
-    userMetadata.put(RestUtils.Headers.USER_META_DATA_HEADER_PREFIX + "key2", "value2");
-    RestUtilsTest.setUserMetadataHeaders(headers, userMetadata);
-    String blobId = postBlobAndVerify(headers, content, refAccount, refContainer);
-
-    headers.put(RestUtils.Headers.BLOB_SIZE, (long) CONTENT_LENGTH);
-    getBlobAndVerify(blobId, null, null, headers, content, refAccount, refContainer);
-    getBlobAndVerify(blobId, null, GetOption.None, headers, content, refAccount, refContainer);
-    getHeadAndVerify(blobId, null, null, headers);
-    getHeadAndVerify(blobId, null, GetOption.None, headers);
-
-    ByteRange range = ByteRange.fromStartOffset(ThreadLocalRandom.current().nextLong(CONTENT_LENGTH));
-    getBlobAndVerify(blobId, range, null, headers, content, refAccount, refContainer);
-    getHeadAndVerify(blobId, range, null, headers);
-
-    range = ByteRange.fromLastNBytes(ThreadLocalRandom.current().nextLong(CONTENT_LENGTH + 1));
-    getBlobAndVerify(blobId, range, null, headers, content, refAccount, refContainer);
-    getHeadAndVerify(blobId, range, null, headers);
-
-    long random1 = ThreadLocalRandom.current().nextLong(CONTENT_LENGTH);
-    long random2 = ThreadLocalRandom.current().nextLong(CONTENT_LENGTH);
-    range = ByteRange.fromOffsetRange(Math.min(random1, random2), Math.max(random1, random2));
-    getBlobAndVerify(blobId, range, null, headers, content, refAccount, refContainer);
-    getHeadAndVerify(blobId, range, null, headers);
-
-    getNotModifiedBlobAndVerify(blobId, null);
-    getUserMetadataAndVerify(blobId, null, headers);
-    getBlobInfoAndVerify(blobId, null, headers);
-    deleteBlobAndVerify(blobId);
-
-    // check GET, HEAD and DELETE after delete.
-    verifyOperationsAfterDelete(blobId, headers, content);
+    for (Container container : refAccount.getAllContainers()) {
+      doPostGetHeadDeleteTest(refAccount, container, refAccount, container);
```

Comment:
Should there be any chance to test on different accounts, which are generated from the same logic?

Reply:
Will add